person.rb

```diff
@@ -1,8 +1,10 @@
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
 
```

Comment:
- [ ] It would be better if we add the `correct_name` method in our person class as we can see in the project requirements
![Screenshot from 2023-10-10 22-34-42](https://github.com/mumbereh/School_Library/assets/46388113/266f53d7-1ce1-4379-a960-e413e887fa11)